portiaui/app/validators/range.js

```diff
@@ -0,0 +1,43 @@
+import { allLetters, allDigits } from '../utils/start-urls';
+
+function hasMixedCase(endpoints) {
+    return endpoints.match(/[a-z]/) && endpoints.match(/[A-Z]/);
+}
+
+function hasSingleLetters(endpoints) {
+    return endpoints.split('-').every((x) => x.length === 1);
+}
+
+function isRangeIncomplete(endpoints) {
+    return endpoints.split('-').contains('');
+}
+
+function validateIncreasing(endpoints, isIncreasing) {
+    const [a, b] = endpoints.split('-');
+    const msg = `Range must be increasing. Try swapping to ${b}-${a}.`;
```

Comment:
`A range must be increasing. Try swapping to ${b}-${a}.`

Just to match the other errors